### ISSUE TITLE
14 days trial

### ISSUE THREAD

**Issue by debian-user-france1** [safing/spn#82]
**What would you like to add or change?**:
For testing

**Why do you and others need this?**:
For testing

**Comment by davegson** [comment]
hey there, thanks for your interest!

I do not believe we will introduce a test tier, since servers cost money and creating a new account is done in a heartbeat. However, since we recently introduced gift cards just send us an email and we might figure something out for you ;)

**Comment by arch-user-france1** [comment]
> hey there, thanks for your interest!
> 
> I do not believe we will introduce a test tier, since servers cost money and creating a new account is done in a heartbeat. However, since we recently introduced gift cards just send us an email, reference this issue and we might figure something out for you ;)

I just bought it... Testing it for a month at the moment. I really miss the feature to bypass the spn for a domain because for example microsoft is really annoying, locking me out all the time, or I wouldn't get a privacy benefit if I use it for sites where I'm logged in like Github.

Maybe a 7 days trial? You could force the user to pay 0.01 one-time and so you could validate every user just by the credit card. Just an idea...

**Comment by davegson** [comment]
> Wait, is the wished feature in SPN rules? I didn't figure that out :)

yes, you can exclude domains in the [SPN Rules](https://docs.safing.io/portmaster/settings#spn/usagePolicy) - or if you need to exclude a full app via the [Use SPN](https://docs.safing.io/portmaster/settings#spn/use) (which can both be configured globally or per-app)

Thanks a lot for your support! <3
